app/views/static_pages/root.html.erb

```diff
@@ -1 +1,10 @@
-<main id="root"></main>
\ No newline at end of file
+<main id="root"></main>
+
+<% if logged_in? %>
+  <script type="text/javascript">
+    window.currentUser = <%= render(
```

Comment:
how does this render work, i dont remember it at all and i cant find any docs on it